src/api/DataHandler.ts

```diff
@@ -1,11 +1,61 @@
 import fetch from 'node-fetch';
 
-const IP = "http://localhost:8081" //"80.167.223.178";
+const api = "localhost:3306" //"80.167.223.178";
```

Comment:
Should probably have a different name, so we know it's actually an IP, if it's API_URL, or API_IP I don't mind.